spec/slack-ruby-bot-server-events/api/endpoints/slack/actions_endpoint_spec.rb

```diff
@@ -48,6 +48,19 @@
         end
       end
 
+      context 'without actions in the payload' do
+        before do
+          payload.delete(:actions)
```

Comment:
Take `actions` out of the payload above, and `merge(actions: ...)` in the existing spec, then you don't have to delete it here, which seems backwards.